### ISSUE TITLE
Not clear how to show terminal hover with keyboard

### ISSUE THREAD

**Issue by roblourens** [microsoft/vscode#188819]
Testing #188537

I can't figure out how to get a terminal hover to show up without using the mouse, so I don't understand how a screenreader user would use this. There is an editor-specific command for this, but maybe there should be one for the workbench? I also don't know how to move focus from the terminal into the tab sidebar

**Comment by meganrogge** [comment]
@roblourens you toggle tab moves focus and use tab / shift tab to get there. 

then, when on the tab, you do `ctrl or cmd + k + i` to open the hover.

**Comment by roblourens** [comment]
Oh yeah, I forgot that my hove keybinding is custom, and it didn't work there, but `cmd+k+i` does
